test/resources/openssl.resource

```diff
@@ -0,0 +1,20 @@
+*** Settings ***
+Documentation       Keywords for using the openssl command line.
+...
+
+Resource            common.resource
+
+
+*** Keywords ***
+Openssl
+    [Documentation]    openssl command wrapper
+    [Arguments]    ${cmd}    ${cmd2}=
+    ${result}=    Run Process    openssl ${cmd} ${cmd2}    shell=True    # robotcode: ignore
+    Should Be Equal As Integers    ${result.rc}    0
+
+Generate CSR Config
+    [Documentation]    Generate csr based on config file
```

Comment:
Please, update documentation or variable names. Do you mean `template_file` is the config file?